### ISSUE TITLE
Make hashing seed a parameter.

### ISSUE THREAD

**Issue by lgautier** [dib-lab/sourmash#76]
Having the hashing seed as parameter could be helpful.

**Comment by ctb** [comment]
On Sun, Jan 01, 2017 at 08:51:23PM -0800, Laurent Gautier wrote:
> Having the hashing seed as parameter could be helpful.

I don't object in principle but I have to admit I'm wondering at the use case :)

**Comment by lgautier** [comment]
I am coming to this because I am having trouble with archives of generated signatures, and while I am not certain of this the seed can be a reason for my trouble.
 
I have two things in mind:
- Allow the generation of hashes for compatibility with signatures generated with a different seed
- Make the seed an explicit parameter (that can be saved when storing signature - something useful when sharing signatures). One could use the hardcoded unchangeable seed but that means that should one get a signature generated with a different seed nothing can be done with.

FWIW I have written a PR (but went into Git nightmare - should have it back some time soon).

**Comment by lgautier** [comment]
Got the pull request back: #79.

**Comment by lgautier** [comment]
The pull request #79 has now a getter for `MinHash` to access the seed (but no setter, since it would not seem of any practical use to change the seed after a `MinHash` is created).

In practice, this is working like follows:

```python
import sourmash_lib._minhash

# create a MinHash with the default seed
mh = sourmash_lib._minhash.MinHash(1000, 21)
assert mh.seed == 42

assert sourmash_lib._minhash.DEFAULT_HASH_SEED == 42

# create a MinHash with a specified seed
mh = sourmash_lib._minhash.MinHash(1000, 21, seed=43)
assert mh.seed == 43
```

**Comment by ctb** [comment]
Closed by #79 and #83.
